src/__tests__/connection/passwordless/__snapshots__/passwordless.test.js.snap

```diff
@@ -1,10 +1,31 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`passwordless connection initPasswordless() calls initNS  with mustAcceptTerms:false when opts.mustAcceptTerms is false 1`] = `
+Array [
+  null,
+  Immutable.Map {
+    "send": "code",
+    "mustAcceptTerms": false,
+  },
+]
+`;
+
+exports[`passwordless connection initPasswordless() calls initNS  with mustAcceptTerms:true when opts.mustAcceptTerms is true 1`] = `
```

Comment:
Extra " "

Reply:
that's just the test runner joining a describe+it. not much I can do